app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,104 @@
+// Place all the styles related to the home controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+body {
+  background-color: #eff3f6;
+  font-family: Helvetica;
+  color: #3f729b;
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+  color: #00405d;
+}
+
+.welcome {
+  font-size: 8em;
+  text-align: center;
+  margin-top: 150px;
+}
+
+.registration {
+  width: 600px;
+  background-color: red;
+}
+
+.form {
+  width: 500px;
+  margin: 100px auto;
+  text-align: left;
+
+}
+
+
+.header {
+  height: 60px;
+  background-color: white;
+  margin-top: -10px;
+  margin-left: -10px;
+  margin-right: -10px;
+  padding: 10px;
+  text-align: center;
+}
+
+.post {
+  width: 275px;
+  height: 60px;
+  line-height: 60px;
+  text-align: left;
+  display: inline-block;
+}
+
+.devise {
+  width: 275px;
+  height: 60px;
+  line-height: 60px;
+  text-align: right;
+  display: inline-block;
+}
+
+.ind_picture {
+  margin: 40px auto 40px;
```

Comment:
Properties should be ordered background-color, border, margin, width<br>Shorthand form for property `margin` should be written more concisely as `40px auto` instead of `40px auto 40px`